spyder/widgets/mixins.py

```diff
@@ -661,6 +661,10 @@ def hide_tooltip(self):
         self._last_point = None
         self.tooltip_widget.hide()
 
+    def reset_tooltip_tip(self):

```

Comment:
`reset_tooltip_tip`?

Is there any other kind of reset, could we just use `reset_tooltip`?

Reply:
Yep `reset_tooltip` everywhere sounds better. I will change it :+1: